compiler/src/iree/compiler/Codegen/Common/GPU/GPUDistributionPatterns.cpp

```diff
@@ -377,13 +377,14 @@ struct DistributeReductions final
                                 vector::CombiningKind combiningKind,
                                 int64_t entriesPerVector, Value mEmpty,
                                 OpBuilder &rewriter, Location loc) const {
-    uint32_t size = maxBitsPerShuffle;
+    // TODO: Get subgroup size from translation info.
+    uint32_t subgroupSize = 64;
```

Comment:
Could we look it up in translation info in this PR? We have some code for this here:
https://github.com/openxla/iree/blob/a86b8bfa9dc6b077b7882fcf88055ef197d1cc8a/compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp#L622-L629

Reply:
I'd prefer that if possible, because as-is, this works on both nvidia and amdgpu. After the current change, the code becomes incompatible with nvidia gpus.

Reply:
yeah +1, reusing this or one of the other utilities for getting subgroup size would be best.

Reply:
Sure I can query the subgroup size from hal.executable.export.